reggie_config/super/init.yaml

```diff
@@ -192,57 +193,61 @@ reggie:
             panels_3: Panels 3 (Woodrow Wilson Ballroom)
-            panels_4: Panels 4 (Annapolis 1-3)
-            panels_5: Panels 5 (National Harbor 14-15)
-            mages_1: MAGES 1 (Chesapeake 7,8,9)
-            mages_2: MAGES 2 (Chesapeake 10,11,12)
-            forum: Forum (Chesapeake H,I)
-            the_forge: Makerspace (Baltimore 3,4)
-            soapbox: The Soapbox (Magnolia 2)
+            panels_4: Panels 4 (Annapolis 2-4)
+            panels_5: Panels 5 (Baltimore 3,4,5)
+            mages_1: MAGES 1 (Chesapeake 4,5,6)
+            mages_2: MAGES 2 (Chesapeake 1,2,3)
+            forum: MAGES Forum (Chesapeake G,H)
+            the_forge: Makerspace (Magnolia 1)
+            soapbox: Makerspace 2 (Magnolia 2)
             autographs: Autographs Red (Expo Hall E)
             autographs_1: Autographs Blue (Expo Hall E)
-            film_fest: MAGES Workshop (Chesapeake 1,2)
+            film_fest: MAGES Workshop & Jam Shop (Chesapeake A,B,C)
             console_stage: Arena (Maryland C)
-            starship_horizons: Starship Horizons (National Harbor 10,11)
-            simulations: Artemis (National Harbor 10,11)
-            console_tournament: Console (Gameroom Tournaments 1)
-            console_attendee: Console (Gameroom Tournaments 2)
-            console_tournament_2: Console (Gameroom Tournaments 3)
-            console_tournament_4: Console (Gameroom Tournaments 4)
-            console_tournament_5: Console (Smash Bros Tournaments)
+            starship_horizons: Simulations - Starship Horizons (National Harbor 10,11)
+            simulations: Simulations - Artemis (National Harbor 10,11)
+            console_tournament: Consoles (Pokemon Events)
+            console_attendee: Consoles (Console Tournaments 1)
+            console_tournament_2: Consoles (Console Tournaments 2)
+            console_tournament_4: Consoles (Fighting Tournaments)
+            console_tournament_5: Consoles (Smash Bros Tournaments)
             arcade_tournaments_1: Arcade (Tournaments 1)
             arcade_tournaments_2: Arcade (Tournaments 2)
             arcade_tournaments_3: Arcade (Tournaments 3)
             arcade_tournaments_5: Arcade (Tournaments 4)
             arcade_tournaments_4: Arcade (Pinball)
-            lan_1: LAN 1
-            lan_2: LAN 2
+            lan_1: LAN 1 (LAN Tournaments)
+            lan_2: LAN 2 (LAN Events)
             lan_theater: LAN Theater
             larp: LARP (National Harbor 14,15)
-            laser_tag: Laser Tag (Chesapeake A,B,C)
-            tabletop_indie: Tabletop Indie Showcase (Riverview Ballroom)
-            tabletop_tournaments: Tabletop Tournaments 1 (Riverview Ballroom)
-            tabletop_tournaments_2: Tabletop Tournaments 2 (Riverview Ballroom)
-            tabletop_panels: Tabletop Discussions (Riverview Ballroom)
+            #laser_tag: Laser Tag (Chesapeake A,B,C)
+            laser_tag: Cosplay Photoshoots (Fountains)
```

Comment:
We re-use this column (and jam_shop, below) because it saves me from having to go in and delete the column in the DB.